### PR TITLE
Contact Us form - validation, Email and Repopulation added

### PR DESCRIPTION
**Pull request by sreekumarmedha**
Added Email functionality with mail trap, form validation for Contact Us form

### CODE REVIEW

File: app/Http/Controllers/ContactController.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                'message' => 'required'
+        ]);
+
+        Mail::send('emails.contact-tmpl', [
+                'name' => $request->get('fname')." ".$request->get('lname'),
+                'email' => $request->get('email'),
+                'phone' => $request->get('phone'),
+                'project' => $request->get('project'),
+                'service' => $request->get('service'),
+                'estimate' => $request->get('estimate'),
```
Comment (manojmedha) [inline app/Http/Controllers/ContactController.php:29]: what is it used for?

File: app/Http/Controllers/ContactController.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                'priority' => $request->get('priority'),
+                'country' => $request->get('country'),
+                'comment' => $request->get('message')
+            ],
+            function ($message) {
+            $message->from('medhadigitalsolutions@gmail.com');
```
Comment (manojmedha) [inline app/Http/Controllers/ContactController.php:35]: my understanding about php  is very minimal, but is it possible to move it to a constant file , like a config so we can avoid hardcoding in code.

File: resources/views/contact.blade.php [hunk 3]
```diff
@@ -29,80 +29,96 @@
...
                                     <div class="input-group">
-                                        <textarea class="form-control">Message</textarea>
+                                        <textarea name="message"  class="form-control" placeholder="Your Message...">{{ old('message') }}</textarea>
                                     </div>
                                 </div>
-                                <div class="col-sm-12">
+                                <!-- <div class="col-sm-12">
                                     <div class="input-group custom-file">
                                         <input type="file" class="form-control custom-file-input">
                                         <label class="custom-file-label">Choose file</label>
```
Comment (manojmedha) [inline resources/views/contact.blade.php:93]: do we need a file uploader? if so , better to set a filtering to allow only specific types to limit security vulnerability

File: app/Http/Controllers/ContactController.php (hunk 2 above)
Comment (sreekumarmedha) [reply app/Http/Controllers/ContactController.php:35]: Yes this is doable.

File: app/Http/Controllers/ContactController.php (hunk 1 above)
Comment (sreekumarmedha) [reply app/Http/Controllers/ContactController.php:29]: Reading the form submit values from http request.

File: resources/views/contact.blade.php (hunk 3 above)
Comment (sreekumarmedha) [reply resources/views/contact.blade.php:93]: We are not planning for file uploads now.

**Pull request closed without merging**